ephemeral-utils.lua

```diff
@@ -0,0 +1,26 @@
+local _Utils = {}
```

Comment:
It's the convention I've seen in example code I've been looking at. The other convention I've seen is to just always call it `_M`. I suspect the conventions will become more obvious as I go.